packages/office-ui-fabric-react/src/utilities/decorators/withViewport.tsx

```diff
@@ -63,6 +63,24 @@ export function withViewport<TProps extends { viewport?: IViewport }, TState>(
       }
     }
 
+    public componentDidUpdate(newProps: TProps) {
+      const { skipViewportMeasures: oldSkipViewportMeasures } = this.props as IWithViewportProps;
+      const { skipViewportMeasures: newSkipViewportMeasures } = newProps as IWithViewportProps;
+      const viewportElement = this._root.current;
+
+      if (oldSkipViewportMeasures !== newSkipViewportMeasures) {
+        if (newSkipViewportMeasures) {
+          this._viewportResizeObserver.disconnect();
+          this._viewportResizeObserver = null;
+          this._events.on(window, 'resize', this._onAsyncResize);
+        } else {
+          this._events.off(window, 'resize', this._onAsyncResize);
+          this._viewportResizeObserver = new (window as any).ResizeObserver(this._onAsyncResize);
```

Comment:
I need to guard this call to ensure `ResizeObserver` is available as we've done in: https://github.com/OfficeDev/office-ui-fabric-react/blob/80119d3d40ca9b185dc6f8f0d3bedfe0296f176d/packages/office-ui-fabric-react/src/utilities/decorators/withViewport.tsx#L54

It is only available in Chromium.

Reply:
Done in https://github.com/OfficeDev/office-ui-fabric-react/pull/9027/commits/476c9a1a87037fdd6d7f8bfbc485320d8e16cf64